src/Components/Tailwind/Toggle.tsx

```diff
@@ -61,13 +61,21 @@ export const Toggle = ({
   );
 };
 
-interface CheckinToggleProps {
+interface CheckStateToggleProps {
   checked: boolean;
   isLoading: boolean;
+  checkState: string;
+  label: string;
   onClick: () => void;
 }
 
-export const CheckinToggle = ({checked, isLoading, onClick}: CheckinToggleProps) => {
+export const CheckStateToggle = ({
```

Comment:
```suggestion
export const CheckInStateToggle = ({
```

And here (you get the idea...).

Reply:
Also here, I didn't add the "In" here because the toggle can now be used to toggle Check-out state, that's why I renamed it